provider/resources.go

```diff
@@ -575,6 +577,8 @@ func Provider() tfbridge.ProviderInfo {
 	}, x.MakeStandardToken(mainPkg)))
 
 	contract.AssertNoErrorf(err, "auto token mapping failed")
+	err = x.AutoAliasing(&prov, prov.GetMetadata())
+	contract.AssertNoErrorf(err, "auto aliasing apply failed")
```

Comment:
Can we add an action item to make sure that code migrations show up in the PR summary, just like bridge and provider updates?